spec/rails_helper.rb

```diff
@@ -592,6 +593,20 @@ class SpecTimeoutError < StandardError
         ActiveRecord::Base.connection.schema_cache.add(table)
       end
 
+      ApplicationController.class_eval do
+        before_action do
```

Comment:
Could do 

```suggestion
      ApplicationController.before_action
```
for less indentation